src/lua/LuaBody.cpp

```diff
@@ -375,6 +381,8 @@ static int l_body_get_position_rel_to(lua_State *l)
  * Parameters:
  *
  *   other - the other body
+ *   (optional) altType - what altitude to calculate (sea-level, above-ground)
+ *   if not specified, sea-level for distant bodies, above-ground otherwise
```

Comment:
If changing the parameters to a LuaBody API function, please also update the function prototype in `data/meta/CoreObject/Body.meta.lua`.

Reply:
Done. Also, argument name is changed from `altType` to `terrainRelative` here, like suggested in the last comment.